pyuvdata/uvflag.py

```diff
@@ -469,20 +1567,41 @@ def combine_metrics(self, others, method='quadmean', inplace=True):
         darray, warray = uvutils.collapse(darray, method, weights=warray, axis=0, return_weights=True)
         this.metric_array = darray
         this.weights_array = warray
-        this.history += 'Combined metric arrays using ' + self.pyuvdata_version_str
+        this.history += 'Combined metric arrays. '
+
+        if not uvutils._check_history_version(this.history, this.pyuvdata_version_str):
+            this.history += this.pyuvdata_version_str
+
+        if run_check:
+            this.check(check_extra=check_extra,
+                       run_check_acceptability=run_check_acceptability)
         if not inplace:
             return this
 
-    def collapse_pol(self, method='quadmean'):
-        """
-        Collapse the polarization axis using a given method.
+    def collapse_pol(self, method='quadmean', run_check=True, check_extra=True,
+                     run_check_acceptability=True):
+        """Collapse the polarization axis using a given method.
 
         If the original UVFlag object has more than one polarization,
         the resulting polarization_array will be a single element array with a
         comma separated string encoding the original polarizations.
 
-        Args:
-            method: How to collapse the dimension(s)
+        Parameters
+        ----------
+        method : str
+            How to collapse the dimension(s). Default is "quadmean"
```

Comment:
Is there a list of the options to use somewhere? Might be good to add the list or point to where to find the list in this docstring.

Reply:
There is an options list in utils it looks like. adding this